anu_lms.module

```diff
@@ -178,3 +180,31 @@ function anu_lms_anu_lms_sw_scripts_alter(&$data) {
     watchdog_exception('anu_lms', $exception);
   }
 }
+
+/**
+ * Implements hook_entity_insert().
+ *
+ * Adds an empty page to a lesson if there are no pages.
+ */
+function anu_lms_entity_insert(EntityInterface $entity) {
+  if ($entity->bundle() !== 'module_lesson') {
```

Comment:
not all entities have bundles as far as I remember. We need to check the entity type first to ensure that it's a node.